service/history/workflow/workflow_task_state_machine.go

```diff
@@ -394,6 +403,9 @@ func (m *workflowTaskStateMachine) AddWorkflowTaskStartedEvent(
 	}
 
 	workflowTask, err := m.ReplicateWorkflowTaskStartedEvent(workflowTask, m.ms.GetCurrentVersion(), scheduleID, startedID, requestID, startTime)
+
+	m.emitWorkflowTaskAttemptStats(workflowTask.Attempt)
```

Comment:
Emit the metrics here so that:
- we will have metric for both succeed and failed (transient) workflow tasks
- we won't have metric for standby namespaces